comp/forwarder/defaultforwarder/forwarder_test.go

```diff
@@ -48,8 +50,14 @@ var (
 	}
 )
 
+func setupconfig() pkgconfigmodel.Config {
```

Comment:
This could probably moved in `pkg/config/setup/test_helpers.go` (or something similar), as this is not forwarder specific and I expect a similar function to be defined in every package using a config mock